doctr/models/builder.py

```diff
@@ -444,14 +470,16 @@ def __call__(  # type: ignore[override]
     def _build_blocks(  # type: ignore[override]
         self,
         boxes: np.ndarray,
+        objectness_scores: np.ndarray,
         word_preds: List[Tuple[str, float]],
         crop_orientations: List[Dict[str, Any]],
     ) -> List[Prediction]:
         """Gather independent words in structured blocks
 
         Args:
         ----
-            boxes: bounding boxes of all detected words of the page, of shape (N, 5) or (N, 4, 2)
+            boxes: bounding boxes of all detected words of the page, of shape (N, 4) or (N, 4, 2)
```

Comment:
Shape was wrong in the docstring ?

Reply:
We detach the scores in the OCRPredictor so afterwards it is (N, 4) / (N, 4, 2) instead of (N, 5) / (N, 4, 2) afterwards .. so before the score was only "dragged along unused"

As mentioned it's unified now with `assume_straight_pages=False | True` :)

Reply:
Yeah, indeed, I understand that after a closer reading of this PR